crates/ark/src/variables/variable.rs

```diff
@@ -1903,6 +1939,14 @@ mod tests {
         })
     }
 
+    #[test]
+    fn test_support_formula() {
+        r_task(|| {
+            let vars = inspect_from_expr("list(x = x ~ y + z + a)");
```

Comment:
Needs a test with multiline deparsing, e.g. with `{`.

Reply:
Fixed and added a test for this!